src/hardware/francaise-instrumentation-ams515/api.c

```diff
@@ -52,14 +52,16 @@ static const uint32_t drvopts[] = {
 
 static const uint32_t devopts[] = {
 	SR_CONF_CHANNEL_CONFIG | SR_CONF_GET | SR_CONF_SET | SR_CONF_LIST,
+	//SR_CONF_OVER_CURRENT_PROTECTION_ENABLED | SR_CONF_GET,
+	//SR_CONF_OVER_CURRENT_PROTECTION_ACTIVE | SR_CONF_GET,
```

Comment:
Why did you comment out the OC options?

Reply:
IIRC I was trying to get SmuView to report the OC condition properly so I tried several ways but it didn't work, always reported on the first channel regardless which was shorted. 
I should try again now that it's not used as a base for my external screen anymore.